src/Tizen.NUI/src/public/Utility/TextPageUtil.cs

```diff
@@ -1,15 +1,15 @@
-using System.Collections.Generic;
 using System;
+using System.Collections.Generic;
+using System.ComponentModel;
 using System.IO;
 
 
 using Tizen.NUI;
 using Tizen.NUI.BaseComponents;
-using Tizen.NUI.Components;
-using Tizen.NUI.Wearable;
 
-namespace Tizen.NUI.Samples
+namespace Tizen.NUI.Utility
 {
+
   class PageData
```

Comment:
please add [EditorBrowsable(EditorBrowsableState.Never)] and some comments even if it is hidden API. ^^;